test/edge-cases.test.js

```diff
@@ -446,6 +446,7 @@ test('client.destroy() - on client with request in progress', function (t) {
   server.listen(function () {
     client = new Client(validOpts({
       serverUrl: 'http://localhost:' + server.address().port
+      // TODO: the _fetchApmServerVersion() here *is* hanging.
```

Comment:
It would be good to deal with this in a separate change. This test case hangs for 30s for the timeout in `_fetchApmServerVersion()`.